ote/src/cljs/ote/views/admin/monitor.cljs

```diff
@@ -71,13 +71,19 @@
     (let [translate-typekw (fn [type]
                               (tr [:enums :ote.db.transport-service/sub-type (keyword type)]))
            companies-by-month-data {:labels (mapv :month (:monthly-companies monitor-data))
-                                    :datasets [{:label "Liikkumispalveluiden tarjoajien lukumäärä"
-                                                :data (mapv :sum (:monthly-companies monitor-data))
-                                                :backgroundColor "rgb(0, 170, 187)"}]}
+                                    :datasets [{:label "Palveluntuottajat"
+                                                :data (mapv :sum-providing (:monthly-companies monitor-data))
+                                                :backgroundColor "rgb(0,136,160)"}
```

Comment:
Hard-coded values without a symbol are a bit hard to maintain. Could these be defined in ote/style and ote/theme

Reply:
These colors are used only in this one bar chart and they are specifically selected only for this one use case. I did try to reuse our colors but the contrast was too small.